### ISSUE TITLE
End Result Predictor in TIC TAC TOE Game

### ISSUE THREAD

**Issue by akash435** [prathimacode-hub/Awesome_Python_Scripts#370]
---
Title: Result Predictor in TIC TAC TOE Game
About: It's a BasicPython script 
Name: Akash Ramanand Rajak
Label: feature request
Assignee: ''

---

Define You:

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

In this script , we allow user to fill the TIC TAC TOE board for with 'X', 'O' and 'empty space'
and there will be a guess button, clicking on which gives the end result of game, like who wins, who lose, or it's a tie.
This script also check that the way user has filled the TIC TAC TOE board, is it possible in real life TIC TAC TOE game.

Also for this, on the starting of the game, it will be clearly given that either person with 'X' will start or person with 'O' will start.

So basically this game is about, predicting the result of the game from any intermediate step (if valid), otherwise if  the  intermediate state that user has filled is not valid, then the script will also lets us know that also.

**Comment by akash435** [comment]
@prathimacode-hub , if we can do something like this in script, then please assign me this under LGM SOC, can add this script.

**Comment by prathimacode-hub** [comment]
Are you adding up an enhancement to the available TicTacToe Game. @akash435

**Comment by akash435** [comment]
no mam , its not a game , it's a script that predict the end result from the intermediate position of TIC TAC TOE board.
so i want to work it as a new issue.

**Comment by prathimacode-hub** [comment]
So you want this to be added as a separate script rather than enhancement? @akash435

**Comment by akash435** [comment]
yes man

**Comment by prathimacode-hub** [comment]
Have separate scripts for tictactoe game might be sounding irrelevant. Instead you can enhance the project to add your script. @akash435

**Comment by akash435** [comment]
Can i do like this, i can give another python file in the same folder of TIC TAC TOE game which is already there and another readme for that

**Comment by prathimacode-hub** [comment]
Ok, sure. That can be done. You can proceed. @akash435

**Comment by akash435** [comment]
Thanks mam
